README.md

```diff
@@ -0,0 +1,118 @@
+
+In order to create your own project from this starter kit using Backlight, go to the [Backlight website](https://backlight.dev/) and sign up, request early access, or log in if you already have an account.
+
+Make sure you have a Backlight Workspace, then go to [simba-starter on backlight](https://backlight.dev/edit/5vtJtbY04aoD1dGKcsu1) and click the **Duplicate** button in the top right corner, this will create a project based on this starter-kit for you.
+
+### NPM
+
+If you want to use simba-starter directly in your application, or extend it without using Backlight, this is possible as simba-starter is published to NPM.
+
+```sh
+npm i @divriots/starter-simba --save
+```
+
+## Contributing
+
+See our [Contribution Guidelines](./CONTRIBUTING.md)
+
+## Rationales
+
+### Types
+
+Lion comes with type definition files and can be used in TypeScript. At the moment, simba-starter does not come with types out of the box just yet, but as this starter-kit progresses, we may add them, either by converting the project to Typescript or by using JSDocs similar to Lion's setup.
+
+### Tests
+
+Lion comes with thousands of tests already, so 95% of the functionality that simba relies on is quite thoroughly unit-tested already. That said, simba adds opinionated extensions on top of lion that are not tested as of now. This will likely be added later and contributions are of course welcome.
+
+### Docs
+
+For documentation, we opt for [MDJS](https://rocket.modern-web.dev/docs/markdown-javascript/overview/), Markdown Javascript. This is markdown files with custom codeblock syntaxes to allow for interactive demos, right in the middle of your written documentation. We believe this is a good way to showcase components, as it is both narrative and live-demo-based. If you use Backlight, `MDJS` integration comes out of the box, so you can go ahead and start writing your MDJS-markdown files straight away without any setup!
+
+### Structure
+
+In the `studio.config.json` we define the structure of this starter-kit, which basically consists of four sections:
+
+- Tokens, values such as colors and spacings, needed to construct and maintain the design system and components
+- Components, lion-based UI components
+- Infrastructure, utilities and mixins used across multiple components.
+- Docs, any utilities used to style the documentation in Backlight or add functionality.
+
+### Importing from @lion
+
+At this moment, simba does not re-export everything from `@lion`, meaning that you might have to import from `@lion` directly, e.g. for `LitElement` or for `localize`. It's important that installation of direct dependencies such as `@lion` or `lit` are properly deduped. In Backlight, `Packd` takes care of this for you on the backend, and you won't have to worry about it. If you use `starter-simba` outside of Backlight, you will need to ensure that deduplication is handled, otherwise you will run into breaking bugs.
```

Comment:
Let's not mention `packd` - we don't communicate about it and I think it's confusing.

I don't think we need to motivate the use of Lion outside of the DS. A DS is a set of components. Every use of Lion should be embedded in the DS.
I would re-export Lion and and note: "To avoid duplication of Lion, you can import from the DS directly instead of pulling another dependency" example...
Something like that.

Reply:
Changed to not talk too much about packd and other confusing things.

Added notes about re-exports existing for lion core modules, but not for the others just yet (due to the bugs I discussed with you on discord), that covers most of the low hanging fruit for now.